### DISCUSSION TITLE
Extracting images from multi-image file

### DISCUSSION THREAD

**Discussion by LGManzoMCB** [ImageMagick/ImageMagick#6198] [Help]
### ImageMagick version

6.6.7-9

### Operating system, version and so on

Windows

### Description

I have an IMG file containing scanned documents captured into one file along with an index file describing the front and rear image offsets and lengths.

My problem is that Magick cannot see beyond the first scanned document in the IMG file. Even a straight convert from IMG to PDF or TIFF, only returns the first page of the first document.

I have tried a series of commands with convert (-verbose, -crop, +adjoin), but all only return the first image in the file.

(I know I have an old version of Magick, but it has worked wonders for me up until now for a variety of projects. This one has me stumped.)
If someone has a suggestion or recommendation, I'd appreciate hearing it.

Thank you!

**Comment by snibgo** [comment]
No "IMG" format is shown on https://www.imagemagick.org/script/formats.php or in "`magick -list format`".

I suggest you link to a sample IMG file, or zip it and upload it to this forum.

**Comment by LGManzoMCB** [comment]
Thank you for the quick response.

Magick treats the IMG file as a TIFF. Here is a sample attempt convert command with the dialogue.

convert -verbose 2Item_20230201_0001.IMG test\test%d.pdf

2Item_20230201_0001.IMG TIFF 1750x700 1750x700+0+0 1-bit Bilevel DirectClass 2.573MB 0.016u 0:00.017

I wish I could provide a sample file, but unfortunately the data is confidential in nature and cannot be released.

Thank you!

**Comment by snibgo** [comment]
Perhaps you can scan some random pieces of paper to make an IMG file you can share.

I suspect your IMG file is not simply a TIFF file with multiple images. What does exiftool say about the file?

Perhaps your IM v6.6.7-9, which is very old, can't read that file correctly.
